src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -281,25 +281,19 @@ protected function describeContainerDefinition(Definition $definition, array $op
         $tableRows[] = array('Public', $definition->isPublic() ? 'yes' : 'no');
         $tableRows[] = array('Synthetic', $definition->isSynthetic() ? 'yes' : 'no');
         $tableRows[] = array('Lazy', $definition->isLazy() ? 'yes' : 'no');
-
-        if (method_exists($definition, 'isSynchronized')) {
-            $tableRows[] = array('Synchronized', $definition->isSynchronized(false) ? 'yes' : 'no');
-        }
+        $tableRows[] = array('Synchronized', $definition->isSynchronized(false) ? 'yes' : 'no');
         $tableRows[] = array('Abstract', $definition->isAbstract() ? 'yes' : 'no');
+        $tableRows[] = array('Autowired', $definition->isAutowired() ? 'yes' : 'no');
 
-        if (method_exists($definition, 'isAutowired')) {
-            $tableRows[] = array('Autowired', $definition->isAutowired() ? 'yes' : 'no');
-
-            $autowiringTypes = $definition->getAutowiringTypes();
-            if (count($autowiringTypes)) {
-                $autowiringTypesInformation = implode(', ', $autowiringTypes);
-            } else {
-                $autowiringTypesInformation = '-';
-            }
-
-            $tableRows[] = array('Autowiring Types', $autowiringTypesInformation);
+        $autowiringTypes = $definition->getAutowiringTypes();
+        if (count($autowiringTypes)) {
```

Comment:
As pointed out multiple times on some other PRs by @nicolas-grekas , the `count` is useless and seems to be avoided thus in the Symfony codebase. Could you take advantage of this PR to change it here too?
Could also be inlined using a ternary without loosing readability here IMHO.